### ISSUE TITLE
Restrict external access?

### ISSUE THREAD

**Issue by GiganteX** [hwdsl2/docker-ipsec-vpn-server#236]
Hi!

Can I (and how?) prevent users accessing external addresses? If I only want them to connect to the VPN for accessing an internal service and absolutely do not want them doing anything else while connected?

**Comment by hwdsl2** [comment]
@GiganteX Hello! This feature is not supported in this Docker image. On a higher level, it will require changes to the IPTables FORWARD rules inside the Docker container, to permit only specific IP addresses.

**Comment by GiganteX** [comment]
You mean I could "docker exec" into the container and modify some settings to make it work? Could you ELI5 how I can accomplish it?

**Comment by hwdsl2** [comment]
First, start a Bash session in the running container:
```
docker exec -it ipsec-vpn-server env TERM=xterm bash -l
```
Example: Assuming that you want to only allow VPN clients to access 10.0.0.0/24 through the VPN, you may add the following IPTables FORWARD rules:
```
iptables -I FORWARD 2 -i ppp+ -o eth0 -d 10.0.0.0/24 -j ACCEPT
iptables -I FORWARD 3 -i ppp+ -o eth0 -j DROP
iptables -I FORWARD 4 -s 192.168.43.0/24 -o eth0 -d 10.0.0.0/24 -j ACCEPT
iptables -I FORWARD 5 -s 192.168.43.0/24 -o eth0 -j DROP
```

Exit the container when finished. Give it a try and see if it works for your use case. If so, you may add these rules to the IPTables section in `/opt/src/run.sh`, to persist after container restarts.
